js/src/dapps/registry/Events/events.js

```diff
@@ -149,6 +221,13 @@ export default class Events extends Component {
             onToggle={ this.onDataChangedToggle }
             style={ inlineButton }
           />
+          <Toggle
```

Comment:
Actually smaller if we didn't have the need to manage toggles. However it is what it is, cannot re-do everything.